src/redux/books/books.js

```diff
@@ -0,0 +1,29 @@
+const ADD_BOOK = 'bookstore/src/redux/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/src/redux/books/REMOVE_BOOK';
```

Comment:
- [ ] Good job so far 👍 The correct way of writing the action type names according to the [ducks pattern](https://github.com/erikras/ducks-modular-redux) is `const ADD_BOOK = 'bookstore/books/ADD_BOOK';` Kindly correct all your action type names 👍

Reply:
Noted thank you. Let me do it right away.